### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shaundon**
Hi, I was checking the contribution guide as I'm using it as inspiration for another project, and noticed some places it could be cleaned up.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -104,11 +104,7 @@ Use `npm run build` to do this.
 
 ### Run the development environment
 
-We use [Storybook](https://storybook.js.org/) for our development environment.
-
-#### Web
-
-Run `npm start` to start the storybook server, then go to [http://localhost:9001](http://localhost:9001) in a web browser to view it.
```
Comment (shaundon) [inline CONTRIBUTING.md:111]: No need to mention web separately as now only web is in this repo.

**Review by olliecurtis — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,15 +126,15 @@ If you want to add a new component, we will need the following:
...
+Figma is the preferred format for non-technical folks. We’d appreciate if you could provide an exact match of your component in Figma format together with folders for each state e.g. disabled, expanded etc.
 
 ### Tokens
 
 Any visual CSS parameters of the component, such as *color, margins, paddings* etc. should not live as magic numbers in the component code, but as **tokens** in the [`@skyscanner/bpk-foundations-web`](https://github.com/Skyscanner/backpack-foundations/packages/bpk-foundations-web) package.
 
 Tokens are defined in the `src/base` directory (with the exception of product-specific tokens, which are in other subdirectories). Tokens come in two layers: In `aliases.json`, all base tokens are defined with concrete values, such as colours, numbers and sizes. The other files then map those aliases to tokens for specific elements.
 
-> You should probably not touch `aliases.json`, as our color palette or grid rarely changes.
+> You should probably not touch `aliases.json`, as our colour palette or grid rarely changes.
```
Comment (georgegillams) [inline CONTRIBUTING.md:137]: tenor-173838902

Comment (shaundon) [reply CONTRIBUTING.md:137]: miss you x

**Comment by olliecurtis** [comment]
Thanks for the external contribution @shaundon we thank you for your OSS contribution

**Pull request merged**